src/IO/H5/Helpers.hpp

```diff
@@ -113,9 +113,43 @@ bool contains_attribute(hid_t file_id, const std::string& group_name,
 
 /*!
  * \ingroup HDF5Group
- * \brief Read a DataVector from a dataset in a group
+ * \brief Open a dataset in an HDF5 group with name "dataset_name"
  */
-DataVector read_data(hid_t group_id, const std::string& dataset_name);
+
+hid_t open_dataset(const hid_t group_id,
+                   const std::string& dataset_name) noexcept;
+
+/*!
+ * \ingroup HDF5Group
+ * \brief Use a dataset to open an HDF5 dataspace
+ */
+
```

Comment:
Nowhere in the code do we have a blank line between the dox comment and the entity's name.